### PR TITLE
feat: modify ui

### PR DESCRIPTION
**Pull request by ssk-wh**
modification:margins, seperatorline width, alignment

Log:

### CODE REVIEW

**Comment by hudeng-go** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-launchpad/pull/103#" title="Author self-approved">ssk-wh</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-launchpad).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-launchpad/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by BLumia — APPROVED** [review]

**Comment by xzl01** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-launchpad/pull/103#pullrequestreview-1927014005" title="Approved">BLumia</a>*, *<a href="https://github.com/linuxdeepin/dde-launchpad/pull/103#" title="Author self-approved">ssk-wh</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-launchpad).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-launchpad/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
